### DISCUSSION TITLE
How to use the rules in the Global Settings page ？

### DISCUSSION THREAD

**Discussion by Tyxiang** [filebrowser/filebrowser#1303] [Q&A]
What is the logical relationship between "Regex" and "Allow"? What is the use of "Insert the path"?

**Comment by o1egl** [comment]
You can read about regular expressions here https://en.wikipedia.org/wiki/Regular_expression. You can define a pattern and set a allow/disallow flag. If the file matches the corresponding pattern, it will be filtered out from the response.
